### PR TITLE
update get-daml-com

### LINKED ISSUE #3915: Automate deployment to get.daml.com
Deployment of https://github.com/digital-asset/daml/blob/master/daml-assistant/get-daml.sh to https://get.daml.com/ should be automated as part of the release process.

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
So far updates to the result of `curl https://get.daml.com` have been made manually. This automates the process.

Fixes #3915.

### CODE REVIEW

File: ci/refresh-get-daml-com.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+             --location \
+             https://get.daml.com \
+         > $tmp
+        if diff daml-assistant/get-daml.sh $tmp; then
+          echo "Files match, nothing to do."
+        else
+          echo "Files differ; pushing new version."
+          aws s3 cp daml-assistant/get-daml.sh s3://get-daml-com/get-daml.sh --acl public-read --region us-east-1
+          echo "Telling Slack..."
+          tell_slack "Change detected in get.daml.com:\`\`\`$(diff daml-assistant/get-daml.sh $tmp)\`\`\`"
```
Comment (carlpulley-da) [inline ci/refresh-get-daml-com.yml:30]: I imagine that doc changes might often be largish? In which case, would a `diff --stat` (to get a summary diff) be better here?

Comment (carlpulley-da) [reply ci/refresh-get-daml-com.yml:30]: Oh, just a moment, we're only looking at `get-dams.sh` changing here - so scrub this comment!

**Review by carlpulley-da — APPROVED** [review]
LGTM!

File: ci/refresh-get-daml-com.yml (hunk 1 above)
Comment (garyverhaegen-da) [reply ci/refresh-get-daml-com.yml:30]: It's just that one file, and I don't expect it to change much, or often. There's an argument to be made for better display (e.g. put the diff in a Slack thread), but I don't think the extra work is worth it in this case.

**Pull request merged**